src/panels/KaitoTestPanel.ts

```diff
@@ -101,6 +101,7 @@ export class KaitoTestPanelDataProvider implements PanelDataProvider<"kaitoTest"
                 const clusterIP = await getClusterIP(this.kubeConfigFilePath, this.modelName, this.kubectl);
                 if (clusterIP === "") {
                     this.isQueryInProgress = false;
+                    vscode.window.showErrorMessage(`Failed to retrieve cluster IP`);
```

Comment:
💡 Idea only, like above: `this.isQueryInProgress = false;` what will this handle post the return?